openshift3/README.md

```diff
@@ -0,0 +1,31 @@
+# Weblate OpenShift
+
+## Provisioning
+
+To upload the weblate template to your current project’s template library, pass the `template.yml` file with the following command:
+
+```bash
+# oc create -f <FILENAME> | <URL>
+oc create -f template.yml
+```
+
+The template is now available for selection using the web console or the CLI.
+
+You can also use the CLI to process templates and use the configuration that is generated to create objects immediately.
+
+```bash
+# oc process -f <FILENAME> | <URL>
+oc process -f https://raw.githubusercontent.com/WeblateOrg/weblate/master/openshift3/template.yml | oc create -f -
+```
+
+## Deprovisioning
+
+```bash
+oc delete all -l app=<APPLICATION_NAME>
+oc delete configmap -l app= <APPLICATION_NAME>
+oc delete secret -l app=<APPLICATION_NAME>
+# ATTTENTION! The following command is only optional and will permanently delete all of your data.
+oc delete pvc -l app=<APPLICATION_NAME>
+
+oc delete all -l app=weblate && oc delete secret -l app=weblate && oc delete configmap -l app=weblate && oc delete pvc -l app=weblate
+```
```

Comment:
Maybe this would better fit into https://github.com/WeblateOrg/weblate/blob/master/docs/admin/install/openshift.rst so that it's rendered in the documentation - https://docs.weblate.org/en/latest/admin/install/openshift.html

Reply:
I know what you mean, personally I thought it was still too less content to list it on the official documentation page, especially regarding the further configuration. However, I wanted to at least provide some minimal information for (experience) users who click on the docs link.

Reply:
I'd really prefer to have it inside the docs rather than in some README. Your instructions are way better than what we have right now. The documentation can be always better, but it's better to progress than to have it split into different locations because some parts are not perfect.

Reply:
Alright, based on the review and possible changes I will try to write a more detailed instruction for the docs next week. And if it's okay for you, I would do this in an extra PR to reach my hacktoberfest goals 🎃

Reply:
Okay, feel free to remove the documentation changes from this PR and submit it separately as addition to existing documentation.